components/Tile.tsx

```diff
@@ -1,6 +1,29 @@
-import { Pane, Badge } from 'evergreen-ui';
+import {
+    Pane, Badge, Button, InfoSignIcon, ArrowTopRightIcon,
+} from 'evergreen-ui';
 
 export default function Tile({ tile }) {
+    // assign colors to badges
+    let col = '';
+    if (tile.type === 'Student Sale') {
+        col = 'red';
+    } else if (tile.type === 'Lost & Found') {
+        col = 'green';
+    } else if (tile.type === 'Bulletin') {
+        col = 'purple';
+    } else {
+        col = 'yellow';
+    }
+    const colData: any = col;
+
+    // Buttons
+    let b1text = '';
+    if (col === 'red') {
```

Comment:
if statement is okay for this one since it is only one case; if there's more than 3 cases use a object/map.